python/lsst/meas/astrom/optimistic_pattern_matcher_b_3D.py

```diff
@@ -0,0 +1,611 @@
+
+from __future__ import division, print_function, absolute_import
+
+from copy import copy
+
+import numpy as np
+from scipy.spatial import cKDTree
+
+__deg_to_rad__ = np.pi/180.0
```

Comment:
Same as before, you can use `np.radians()` and `np.degrees()` to avoid an extra global.